### PR TITLE
Add API endpoint 

### LINKED ISSUE #22: Need an API end point
We are working on an app to fetch all the data of people who need help with click of a single button, thinking of merging with the rescue kerala website, Can we get an API end point for the same , in case if we are pushing details to this server from the mobile app ?

### PR DESCRIPTION
**Pull request by kurianbenoy**
Hopefully fixes #22

### CODE REVIEW

File: floodrelief/settings.py [hunk 1]
```diff
@@ -185,3 +189,9 @@ def get_list(text):
 )
 
 ADMIN_SITE_HEADER = "Keralarescue Dashboard"
+
+CORS_ORIGIN_WHITELIST = (
```
Comment (abdullatheef) [inline floodrelief/settings.py]: Please move this to .env

**Comment by abdullatheef** [comment]
It seems migration file is empty.

File: floodrelief/settings.py (hunk 1 above)
Comment (kurianbenoy) [reply floodrelief/settings.py]: I dont think @abdullatheef  its necessary. It again become more difficult for beginner contributors to contribute . Also django-environ doesn't recommend adding all the setting confiugrations to .env file. It's meant to be for specific things like databases,caches,secret key,debug

**Comment by abhijithanilkumar** [comment]
Why have you committed migrations?

**Comment by kurianbenoy** [comment]
@abhijithanilkumar please check main migrations:
https://github.com/IEEEKeralaSection/rescuekerala/tree/master/mainapp/migrations

I didn't want to add  migrations initially . Some people here requested to add migrations

**Comment by kurianbenoy** [comment]
Can someone test this API

**Pull request closed without merging**